src/components/filters/getFilterProps.js

```diff
@@ -207,8 +235,17 @@ function getRangeFilterProps(column, descriptions, filters, setFilter) {
       confirm()
   }
 
+  const minRangeValue = value => {
+    return value?.[0]?.min
+  }
+
+  const maxRangeValue = value => {
+    return value?.[0]?.max
+  }
+
```

Comment:
Same comment here. We can just assign `const minValue = ...` instead of defining a function for it.